runtime-light/stdlib/crypto/crypto-functions.cpp

```diff
@@ -115,3 +110,175 @@ task_t<int64_t> f$openssl_verify(const string &data, const string &signature, co
 
   co_return 1;
 }
+
+Optional<tl::CipherAlgorithm> parse_algorithm(const string &method) {
+  if (strncasecmp(method.c_str(), "aes-128-cbc", strlen("aes-128-cbc")) == 0) {
+    return tl::CipherAlgorithm::AES128;
+  } else if (strncasecmp(method.c_str(), "aes-256-cbc", strlen("aes-256-cbc")) == 0) {
+    return tl::CipherAlgorithm::AES256;
+  }
+  return {};
+}
+
+int64_t algorithm_iv_len(tl::CipherAlgorithm algorithm __attribute__((unused))) {
```

Comment:
We are in C++20, so `[[maybe_unused]]` is a perfect choice here

Reply:
fix